src/screen.py

```diff
@@ -24,12 +26,36 @@ def __init__(self, monitor: int = 0, wait: int = 20):
         self._config = Config()
         self._monitor_roi = self._sct.monitors[monitor_idx]
         # auto find offests
-        template = load_template(f"assets/templates/main_menu_top_left.png", 1.0)
-        template_ingame = load_template(f"assets/templates/window_ingame_offset_reference.png", 1.0)
-        start = time.time()
         self.found_offsets = False
         Logger.info("Searching for window offsets. Make sure D2R is in focus and you are on the hero selection screen")
+        position = self.find_window_via_winapi() if self._config.general["find_window_via_win32_api"] else None
+        if position is None:
+            position = self.find_window_via_assets(wait)
+        if position is not None:
+            self._set_window_position(*position)
+        else:
+            if self._config.general["info_screenshots"]:
```

Comment:
I think I spent like 10 minutes reading these lines, I think it would be clearer to write something like this:

`
position = self.find_window_via_winapi() if self._config.general["find_window_via_win32_api"] else self.find_window_via_assets(wait)

if position is not None:
  ...
`

that should have the same outcome no?

Reply:
Not really. self.find_window_via_winapi() will return None in case it can't find the window using WinAPI. So in the code you're suggesting we won't have a fallback in case WinAPI fails. But let me do a change here to get this more readable and also fix misleading logging information

Reply:
I've pushed an update. It should be more readable now I think with log messages reflecting the process